lib/config.js

```diff
@@ -34,20 +18,16 @@ var getSysConfRoot = function() {
  * @TODO: We might want to either expand the version checks or maybe do a lower
  * level check of the process file descriptors
  */
-var processType = function() {
-
-  // Collect some info on things we have
-  var isElectron = _.has(process, 'versions.electron', false);
-  var isChrome = _.has(process, 'versions.chrome', false);
-  var isAtom = _.has(process, 'versions.atom-shell', false);
-
-  // If any of the above are true return the type
-  return (isElectron || isChrome || isAtom) ? 'browser' : 'node';
+const isBrowser = () =>  _(process.versions)
+  .reduce((isBrowser, version, thing) => (isBrowser || _.includes(browsers, thing)), false);
```

Comment:
<img src=https://media.tenor.com/images/97df88a327e0805190a7facedc25f39b/tenor.gif />

Reply:
@pirog I <3 this functional flow, however, it could possibly use a bit more commenting/documentation It looks to me like this might be some recursion going on? Reading the lodash docs on reduce give me sort of an idea, and I may just be missing some context by looking at this on Github as a diff